### PR TITLE
Crafting update episode 1

### PR DESCRIPTION
**Pull request by Aineias1**
## About the PR

Added bunch of new craftable stuff, and items to craft with
## Why / Balance

milestone
## Technical details

7
## Media

7
## Requirements

- [x ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Aineias1** [comment]
Gonna continue tomorrow, and will try to add a craft for it

File: Resources/Prototypes/_Goobstation/Entities/Objects/Tools/jetpacks.yml [hunk 1]
```diff
@@ -33,3 +33,39 @@
...
+      sprite: _Goobstation/Objects/Tanks/improvised.rsi
+    - type: Sprite
+      sprite: _Goobstation/Objects/Tanks/improvised.rsi
+    - type: Clothing
+      sprite: _Goobstation/Objects/Tanks/improvised.rsi
+      slots:
+        - Back
+        - suitStorage
+    - type: GasTank
+    outputPressure: 42.6
```
Comment (a-person5660) [inline Resources/Prototypes/_Goobstation/Entities/Objects/Tools/jetpacks.yml:55]: pretty sure this has to be indented

**Comment by Aineias1** [comment]
okay, fixed issue with textures but the game is still broken

**Comment by Aidenkrz** [comment]
You add improvised jetpacks but no construction graphs?

**Comment by Aineias1** [comment]
> You add improvised jetpacks but no construction graphs?

i didn't make them yet :godmode: 
i WILL make them

**Comment by Aineias1** [comment]
image

**Comment by Aidenkrz** [comment]
if the pr isnt finished dont make it open

**Comment by Aidenkrz** [comment]
hasnt been touched in two weeks, ask for it to be reopened when you are working on it again / its done

**Pull request closed without merging**